DataCapturing/DataCapturingService.swift

```diff
@@ -69,22 +69,19 @@ public class DataCapturingService: NSObject {
     }()
 
     /**
-     An API to store, retrieve and update captured data to the local system until the App
+     The *CoreData* stack used to store, retrieve and update captured data to the local system until the App
      can transmit it to a server.
      */
-    let persistenceLayer: PersistenceLayer
```

Comment:
Sharing one persistence layer between tracks leads to an ugly race condition. I removed that by requiring the creation of a new persistence layer on demand. To create such a persistence layer one requires a CoreDataManager, which therefore now is an attribute. Sharing a CoreDataManager between threads is possible.